x-pack/plugins/observability/public/pages/alert_details/mock/index.ts

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
nit: I think if we rename this file to `alert` instead of `index` it will be easier to understand what is the content.

Reply:
makes sense. I will rename the file.